src/main/resources/fabric.mod.json

```diff
@@ -37,7 +40,8 @@
     "java": ">=17",
     "cloth-config2": ">=6.2.57",
     "fabricloader": ">=0.13.3",
-    "minecraft": "1.18.2"
+    "minecraft": "1.18.2",
+    "voicechat": ">=1.18.2-2.2.39"
```

Comment:
Why do you put SVC in the depends category ?
I don't want to use it and i have to use an override to make my game launch.
Shouldn't the suggest category be a possibility ?

Reply:
Fixed this :)